retriever/engines/mysql.py

```diff
@@ -19,6 +21,8 @@ class engine(Engine):
         "bool": "BOOL",
     }
     max_int = 4294967295
+
+    sql_lookup = {}
```

Comment:
I think this should be with the addition of other comments below
self.sql_lookup = {"user": "root", "password": ""]}
or 
self.sql_lookup = {"user": required_opts[0][3], "password": required_opts[1][3]}
We set them as the defaults in the dictionary